### PR TITLE
.NET 6 Update

### PR DESCRIPTION
**Pull request by CaptnCodr**
- Adds general net6.0 support
- Drops F# 4.7.x language support
- Updates test projects to net6.0

### CODE REVIEW

File: paket.dependencies [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source https://nuget.org/api/v2
-framework: netstandard2.0, net5.0
+framework: netstandard2.0, net5.0, net6.0
```
Comment (sergey-tihon) [inline paket.dependencies:2]: we can remove net5.0 from here

**Comment by sergey-tihon** [comment]
you also need to run `dotnet paket install` to properly update lock file

**Comment by CaptnCodr** [comment]
Thanks Sergey! 🙂 
I also refactored some strings with string interpolation in some operators.
Do you want more changes? Release as 4.1.0 or 5.0.0?

**Comment by sergey-tihon** [comment]
No, looks good! 
Version is up to you. We still build for netstandard2.0 and there should not be any breaking changes.

**Comment by CaptnCodr** [comment]
So, then I'll publish it as 4.1.0 soon.

**Pull request merged**